webpack/assets/javascripts/react_app/components/SearchBar/__tests__/integration.test.js

```diff
@@ -35,11 +36,9 @@ describe('SearchBar integration test', () => {
       .find('.autocomplete-search-btn')
       .first()
       .simulate('click');
-    // expect it to call Turbolinks.
-    expect(global.Turbolinks.visit.mock.calls).toHaveLength(1);
+    expect(global.window.location.href).not.toEqual(oldHref);
     const event = new KeyboardEvent('keypress', { charCode: KEYCODES.ENTER });
     global.dispatchEvent(event);
-    expect(global.Turbolinks.visit.mock.calls).toHaveLength(2);
```

Comment:
Shouldn't we expect something after pressing ENTER?

Reply:
I don't know what to expect though :thinking:

Reply:
Would it make sense spying on the `global.window.location.href` setter?
https://jestjs.io/docs/en/jest-object#jestspyonobject-methodname-accesstype

Reply:
I needed a bit of codewrangling. Not sure if this is the best way, but it works :thinking: :laughing:

Reply:
LGTM :+1: